### PR TITLE
Prepare demo Calling Escrow Success

### PR DESCRIPTION
**Pull request by ShellyWEI**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior?** (You can also refer to a JIRA ticket here)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **What dependencies / modules need to be updated as pre-requisites?** (Include relevant PR links here)

* **Description of changes**

---

* **Please check if the PR fulfills these requirements**
- [ ] The subject of this PR contains a JIRA ticket BTFS-xxxx (N/A for community PRs)
- [ ] PR description and commit messages are [good and meaningful](https://chris.beams.io/posts/git-commit/)
- [ ] Unit tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **Please make sure the following procedures have been applied before requesting reviewers**
- [ ] Code changes closely follow [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
- [ ] Code has been *rebased* against latest master (`git merge` not recommended, unless you know what you are doing)
- [ ] Code changes have run through `go fmt`
- [ ] Code changes have run through `go mod tidy`
- [ ] All unit tests passed locally (`make test_go_test`)

### CODE REVIEW

File: cmd/btfs/daemon.go [hunk 1]
```diff
@@ -324,8 +323,8 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 	hValue, _ := req.Options[hValueKwd].(string)
 
 	// Btfs auto update.
-	url := fmt.Sprint(strings.Split(cfg.Addresses.API[0], "/")[2], ":", strings.Split(cfg.Addresses.API[0], "/")[4])
-	go update(url, hValue)
+	//url := fmt.Sprint(strings.Split(cfg.Addresses.API[0], "/")[2], ":", strings.Split(cfg.Addresses.API[0], "/")[4])
+	//go update(url, hValue)
```
Comment (taiyangc) [inline cmd/btfs/daemon.go]: uncomment before you merge.

File: core/commands/storage.go [hunk 2]
```diff
@@ -213,7 +212,7 @@ Receive proofs as collateral evidence after selected nodes agree to store the fi
 			return fmt.Errorf("price is smaller than minimum setting price")
 		}
 		if !found {
-			price = 10
+			price = 250000
```
Comment (taiyangc) [inline core/commands/storage.go]: add a TODO here. needs to come from hub

File: core/hub/sync.go [hunk 3]
```diff
@@ -47,7 +47,8 @@ func QueryHub(ctx context.Context, node *core.IpfsNode, mode string) ([]*hubpb.H
 	err = grpc.HubQueryClient(config.Services.HubDomain).WithContext(ctx, func(ctx context.Context,
 		client hubpb.HubQueryServiceClient) error {
 		resp, err = client.GetHosts(ctx, &hubpb.HostsReq{
-			Id: node.Identity.Pretty(),
+			Id:       node.Identity.Pretty(),
+			RespSize: 30,
```
Comment (taiyangc) [inline core/hub/sync.go]: The goal was to grab the default (100) from hub, why set 30 here?

**Comment by taiyangc** [comment]
3 comments from me, rest looks ok

File: core/hub/sync.go (hunk 3 above)
Comment (ShellyWEI) [reply core/hub/sync.go]: i can unset, it only for early stages when hub returning hosts not consistent with set hosts

File: core/commands/storage.go (hunk 2 above)
Comment (ShellyWEI) [reply core/commands/storage.go]: it's coming from hub see line 246

File: cmd/btfs/daemon.go (hunk 1 above)
Comment (ShellyWEI) [reply cmd/btfs/daemon.go]: ok

File: core/commands/storage.go (hunk 2 above)
Comment (taiyangc) [reply core/commands/storage.go]: then, you should set `price = HostPriceLowBoundary`

Comment (taiyangc) [reply core/commands/storage.go]: I mean, whatever the default value is, it shouldn't be a magic number

File: core/hub/sync.go (hunk 3 above)
Comment (ShellyWEI) [reply core/hub/sync.go]: no I can't I tested if I setting to 100, the order returning from hub is not what I am setting here

**Pull request merged**